src/test/regress/expected/columnar_chunk_filtering.out

```diff
@@ -23,6 +23,7 @@ $$
         RETURN result;
     END;
 $$ LANGUAGE PLPGSQL;
+set columnar.qual_pushdown_correlation = 0.0;
```

Comment:
should we have a few simple tests where `qual_pushdown_correlation` is not set to `0.0` ?

Reply:
Done, I think.